ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/AbstractBlockProcessor.java

```diff
@@ -156,68 +150,62 @@ protected AbstractBlockProcessor(
       final List<Transaction> transactions,
       final List<BlockHeader> ommers,
       final PrivateMetadataUpdater privateMetadataUpdater) {
-    final Span globalProcessBlock =
-        tracer.spanBuilder("processBlock").setSpanKind(Span.Kind.INTERNAL).startSpan();
-    try {
```

Comment:
so tracer has moved to JsonRpcHttpService - but the rest of this change I think is unrelated - is this a separate refactor?

Reply:
I removed the tracer here. The tracer would start before metrics system was ready, creating issues. And this trace didn't seem very interesting when instrumented, so might as well remove it.

Reply:
Yes, the try/catch is removed so the indentation changed